### ISSUE TITLE
Slides with interactive elements fail accessibility when off screen

### ISSUE THREAD

**Issue by allypalanzi** [FormidableLabs/nuka-carousel#821]
# Bugs and Questions

### Prerequisites

**Feel free to delete this section if you have checked off all of the following.**

- [x] I've searched open [issues](https://github.com/FormidableLabs/nuka-carousel/issues) to make sure I'm not opening a duplicate issue
- [x] I have read through the docs before asking a question
- [x] I am using the latest version of nuka-carousel

### Describe Your Environment

- What version of nuka-carousel are you using? 4.8.2
- What version of React are you using? 17.0.2
- What browser are you using? Chrome

### Describe the Problem
If a slide contains interactive elements, like a button, link, etc, users are still able to tab to the slides when the slides are off screen and aria-hidden.

![Screen Shot 2021-12-08 at 11 24 22 AM](https://user-images.githubusercontent.com/1761730/145254713-172fad84-c183-4810-9e30-3e5df00544e9.png)

https://codesandbox.io/s/ecstatic-wiles-l2p8k?file=/src/index.js
You can see here that a user can still tab to the slide that is off screen.

**Expected behavior:** Interactive elements within slides have tabIndex=-1 applied to them when inside of a slide that is not visible and aria-hidden

**Actual behavior:** Off screen slides can still be tabbed to even when aria-hidden

**Comment by ValGeorgiev** [comment]
Hey @allypalanzi thanks for raising this with us. I will include this issue to the roadmap for nuka v5.
